### ISSUE TITLE
spawn configs for multiplayer servers 

### ISSUE THREAD

**Issue by HorkDorkkus** [EterDelta/CrittersAndCompanions#91]
hi ive noticed on a multiplayer server 4 threads 10 gb ram  a big bit of lag is coming from some mobs (otters dragonflies) and some are just more than i want them would it be possible to add a configuration file that will allow you set some sort of limit whether it be per player ,worlds, or server 
![image](https://github.com/user-attachments/assets/d0e28e93-59b9-4597-8118-29b57bde105b)

**Comment by PssbleTrngle** [comment]
Which version is this happening on?

**Comment by HorkDorkkus** [comment]
Mod version 2.12.22

**Comment by NikzardGG** [comment]
same, the mod without a config file = uninstall

**Comment by Corosauce** [comment]
Same problem here, mod has zero config options I guess. Will have to resort to McJtys In Control

**Comment by CaptainMinnette** [comment]
Agreed. There seems to be an issue where dragonflies spawn in, fly out of range of players, and become unloaded in that chunk. We have a few chunks full to the gills with dragonflies. A proper server config file might help.
![Critters and Companions Dragonflies Issue](https://github.com/user-attachments/assets/568a1f73-327d-44ac-b5cf-fb2f03f03a37)

**Comment by PssbleTrngle** [comment]
Same as I mentioned in #95, this mod has no config file to configure the spawns, because they are (at least on forge) defined using data files, which can be overwritten using a datapack.

**Comment by PssbleTrngle** [comment]
I will look into why with the default spawn rates there are some many dragonflys tho

**Comment by PssbleTrngle** [comment]
The dragonfly spawn weight has been decreased a bit. Spawns can now also be configured on fabric using a config file & on forge using a datapack (see #95)
